ouroboros-consensus-cardano/changelog.d/20230815_110046_joris_275_consolidate_protocolparams_across_eras.md

```diff
@@ -0,0 +1,33 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+
+### Non-Breaking
```

Comment:
Huh. It hadn't occurred to me that you could have multiple of these sections. But I guess `scriv` does call them "categories", which suggests "tags", which suggests multiplicity.